src/components/Header.stories.js

```diff
@@ -0,0 +1,11 @@
+import Header from './Header';
+
+export default {
+  title: 'Component/Header',
+  component: Header,
+};
+
+const Template = args => <Header {...args} />;
+
+export const Default = Template.bind({});
+Default.args = {};
```

Comment:
If you do not have any args, you can also write it like that, if you want:

```suggestion
export const Default = () => <Header /> ;
```